node/cmd/guardiand/node.go

```diff
@@ -582,8 +579,7 @@ var Build = "prod"
 // initConfig initializes the file configuration.
 func initConfig(cmd *cobra.Command, args []string) error {
 	return node.InitFileConfig(cmd, node.ConfigOptions{
-		FilePath:  configPath,
-		FileName:  configFilename,
+		FilePath:  viper.ConfigFileUsed(),
```

Comment:
please update the [operations document](https://github.com/wormhole-foundation/wormhole/blob/main/docs/operations.md#config-file) either in this PR or a follow up.